Civi/RemoteToolsRequest.php

```diff
@@ -346,9 +346,12 @@ public function wasExecuted()
      */
     public function getOriginalReturnFields()
     {
-        $return_string = \CRM_Utils_Array::value('return', $this->original_request, '');
-        if ($return_string) {
-            return explode(',', $return_string);
+        $return = $this->original_request['return'] ?? '';
```

Comment:
`$this->original_request['return'] ?? []` would avoid the `explode()` if `return` is not set or null.